packages/react-components/react-calendar-compat/stories/Calendar/CalendarContiguousWork.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { Calendar, DateRangeType, DayOfWeek } from '@fluentui/react-calendar-compat';
+
+const workWeekDays = [DayOfWeek.Monday, DayOfWeek.Tuesday, DayOfWeek.Wednesday, DayOfWeek.Thursday, DayOfWeek.Friday];
```

Comment:
Should there be constants defined in the component for common work week patterns (e.g. M-F, M-Su, Su-Th) ?